floor/lib/src/adapter/insertion_adapter.dart

```diff
@@ -87,4 +87,24 @@ class InsertionAdapter<T> {
     }
     return result;
   }
+  
+    Future<void> _insertListVoid(
+      final List<T> items,
+      final ConflictAlgorithm conflictAlgorithm,
+      ) async {
```

Comment:
🔍 Formatting seems to be off here
```suggestion
  Future<void> _insertListVoid(
    final List<T> items,
    final ConflictAlgorithm conflictAlgorithm,
  ) async {
```